docs/mom/files.md

```diff
@@ -0,0 +1,16 @@
+# [Kingdom Hearts Melody of Memory](index.md) File Overview
+
+## Extensions
+| Name | Purpose |
+|--|--|
+| .bytes | Denotes files that Unity does not natively support, can be ignored or removed when working on the files in question, be sure to put it back when done |
+| [.sed](sed.md) | Usually has .bytes after it, proprietary Square audio format. Also used in Final Fantasy XV for the Regalia radio tracks |
+
+## Directory Layout
+<pre>
+|-- Data
+|   |-- StreamingAssets
+|   |   |-- <a href="assetbundle.md">AssetBundles</a>
```

Comment:
Avoid to use HTML in markdown when possible. Use the syntax `[AssetBundles](assetbundle.md)` when possible.